.ci-local/credentials.sh

```diff
@@ -68,3 +68,11 @@ org.librarysimplified.nexus.password=${NYPL_NEXUS_PASSWORD}
 org.librarysimplified.app.assets.openebooks=${OPENEBOOKS_CREDENTIALS}
 org.librarysimplified.app.assets.simplye=${SIMPLYE_CREDENTIALS}
 EOF
+
+#------------------------------------------------------------------------
+# Addding slack webhook to environment
+SLACK_WEBHOOK_URL=$(</Users/medwar40/Workspace/NYPL/Certificates/SimplyE/slack-webhook.url) || 
```

Comment:
Fairly sure this line isn't going to work on CI!

The right path would be `.ci/credentials/SimplyE/slack-webhook.url`.

Reply:
Ooops, nice catch!